core/src/apps/debug/load_device.py

```diff
@@ -44,9 +44,8 @@ async def load_device(ctx, msg):
         needs_backup=msg.needs_backup is True,
         no_backup=msg.no_backup is True,
     )
-    storage.device.load_settings(
-        use_passphrase=msg.passphrase_protection, label=msg.label
-    )
+    storage.device.set_passphrase_enabled(msg.passphrase_protection)
+    storage.device.set_label(msg.label or "")
```

Comment:
I am thinking maybe `or ""` should be handled inside `set_label`?

Reply:
Do you think that `set_label` should accept `Optional[str]`? Why?

Reply:
I take that back. Ignore, you are right.